### PR TITLE
Future/add book

### PR DESCRIPTION
**Pull request by KarolGin**

### CODE REVIEW

File: src/pages/AddProduct/AddProduct.tsx [hunk 1]
```diff
@@ -1,12 +1,109 @@
...
-export const addProduct = () => {
+  const addProduct = async () => {
+    try {
+      const res = await fetch("https://fakeapi.extendsclass.com/books", {
+        method: "POST",
+        headers: { "Content-Type": "application/json",
+         },
 
+        body: JSON.stringify({
+          title: title,
```
Comment (szu-szy) [inline src/pages/AddProduct/AddProduct.tsx:23]: if you have the same name of prop and variable you can do it shorter > title: title > title, the same below isbn etc.

**Review by szu-szy — COMMENTED** [review]
please format files

File: src/pages/AddProduct/AddProduct.tsx (hunk 1 above)
Comment (KarolGin) [reply src/pages/AddProduct/AddProduct.tsx:23]: How can I write this shorter? I have a title: title so I don't understand the mistake :(

Comment (szu-szy) [reply src/pages/AddProduct/AddProduct.tsx:23]: body: JSON.stringidy({
title, 
isbn,
...

Comment (KarolGin) [reply src/pages/AddProduct/AddProduct.tsx]: Done

**Review by szu-szy — APPROVED** [review]

**Pull request merged**